lib/Fhp/Options/FinTsOptions.php

```diff
@@ -45,6 +45,14 @@ class FinTsOptions
     /** @var int */
     public $timeoutResponse = 30;
 
+    /**
+	 * The Kundensystem-Id as returned by the bank and persisted by the application code
+	 * Prevents having to re-authenticate every time on login
+	 * Use DialogInitialization::getKundensystemId() on the return-object of FinTs::login(), to get the new one
```

Comment:
Nit: "return object" without dash (compound nouns in English don't use dash)

My actual question: Why does it say "the new one"? Isn't that the "only" one?

I see two potential misunderstandings (of a library user) here:
* Am I supposed to run `login()` every time and then get the kundensystemId?
* After getting the kundensystemId, do I need to put it into FinTsOptions right away? And then what, create a new FinTs object right away?

Probably it would help to explain how this interacts with persistence, namely that it's meant to be stored in the database and be used _the next time_ a FinTs instance is created, in a new PHP run. And that it's redundant if full `persist()` is used.